src/widgets/NftDetailWidget/NftDetailWidget.tsx

```diff
@@ -7,11 +7,15 @@ import NftDetailMainInfo from './subcomponents/NftDetailMainInfo/NftDetailMainIn
 
 import './NftDetailWidget.scss';
 
-const CollectionWidget: FC = () => {
+interface CollectionWidgetProps {
+  className?: string;
+}
+
+const CollectionWidget: FC<CollectionWidgetProps> = ({ className = '' }) => {
   const { collectionImage } = useAppSelector((state) => state.config);
 
   return (
-    <div className="nft-detail-widget">
+    <div className={`nft-detail-widget ${className}`}>
       <div className="nft-detail-widget__top">
         <NftDetailMainInfo
           subTitle="by sjnivo"
```

Comment:
Don't commit this, it'll make the auto merge for my branch have merge conflicts and I'm keen to prevent manual conflict fixing :smile:

Reply:
Sure, I'll remove this temporarily.